apicapi/apic_client.py

```diff
@@ -335,6 +335,27 @@ class ManagedObjectClass(object):
                 'rsvswitchExporterPol-[uni/infra/vmmexporterpol-%s]'),
         'netflowVmmExporterPol': ManagedObjectName('infraInfra',
                                                    'vmmexporterpol-%s'),
+
+        # ERSPAN
+        'spanVSrcGrp': ManagedObjectName('infraInfra', 'vsrcgrp-%s'),
+        'spanVSrcGrp': ManagedObjectName('fvTenant', 'vsrcgrp-%s'),
```

Comment:
Does this actually work? I see other similar cases use a different key. eg. fvRsCons vs fvRsCons__Ext when a class has multiple possible parents in the containment hierarchy

Reply:
You are right, doubleunderscore is stripped off when handling these keys. Will address that in the latest commit